errors/errors.go

```diff
@@ -13,13 +13,17 @@ const (
 	MsgErrInvalidParam      = "invalidParameter"
 	MsgErrOpNotPermitted    = "operationNotPermitted"
 	MsgErrDisabledEndpoint  = "disabledEndpoint"
-
-	AuthHeader   = "authorizationHeader"
-	BasicToken   = "basicToken"
-	BearerToken  = "bearerToken"
-	Token        = "token"
-	Level        = "level"
-	JSONExpected = "JSONExpected"
+	MsgErrInvalidLength     = "invalidLength"
+	MsgErrUnknown           = "unknowError"
+
+	EncryptDecrypt = "encrypt/decrypt"
```

Comment:
I know you copied/pasted from another project but I don't know if it is safe to use a slash?

Reply:
encryptOrDecrypt seems safer

Reply:
OK